README.md

```diff
@@ -55,11 +55,12 @@ You can log any issues or suggestion related to this library [here](https://gith
 
 Read our contributing [guide](https://github.com/arshadkazmi42/currency-symbols/blob/master/CONTRIBUTING.md) on getting started with contributing to the codebase
 
-## Contributors
+<!---  ## Contributors
```

Comment:
Can we just remove the whole block. We don't want it to be commented. Lets just remove it completely.